backend/de.metas.business/src/main/java/de/metas/costing/methods/AveragePOCostingMethodHandler.java

```diff
@@ -151,7 +152,10 @@ protected CostDetailCreateResult createOutboundCostDefaultImpl(final CostDetailC
 
 	private CostDetailCreateResult createCostDetailAndAdjustCurrentCosts(final CostDetailCreateRequest request)
 	{
-		final boolean isInboundTrx = request.getQty().signum() >= 0;
+		final Quantity requestQty = request.getQty();
```

Comment:
i think requestQty is not used

Reply:
I am using it now. Same as requestAmt. I think it's easier to read.